packages/mdc-top-app-bar/index.ts

```diff
@@ -107,22 +86,21 @@ class MDCTopAppBar extends MDCComponent {
           this.navIcon_.removeEventListener(evtType, handler);
         }
       },
-      notifyNavigationIconClicked: () => {
-        this.emit(strings.NAVIGATION_EVENT, {});
-      },
-      registerScrollHandler: (handler) => this.scrollTarget_.addEventListener('scroll', handler),
-      deregisterScrollHandler: (handler) => this.scrollTarget_.removeEventListener('scroll', handler),
+      notifyNavigationIconClicked: () => this.emit(strings.NAVIGATION_EVENT, {}),
+      registerScrollHandler: (handler) => this.scrollTarget_.addEventListener('scroll', handler as EventListener),
```

Comment:
I don't think you need to cast them to EventListeners here.

Reply:
Removing them throws an error:

```
[tsl] ERROR in packages/mdc-top-app-bar/index.ts(92,89)
      TS2345: Argument of type 'SpecificEventListener<"scroll">' is not assignable to parameter of type 'EventListener | EventListenerObject | null'.
  Type 'SpecificEventListener<"scroll">' is not assignable to type 'EventListener'.
    Types of parameters 'evt' and 'evt' are incompatible.
      Type 'Event' is missing the following properties from type 'UIEvent': detail, view, initUIEvent
```

That's because `scrollTarget_` is an `EventTarget` rather than an `Element`, and `EventTarget` doesn't have generic signatures for `addEventListener()` or `removeEventListener()` in `lib.dom.d.ts` (whereas `Element` does).

Reply:
I see - its because we need to set scrollTarget to EventTarget - mainly because it could be the Window.